Module-generation/frontmatter.py

```diff
@@ -1,11 +1,8 @@
 import os
 import json
 
-
-
 DIR="./meta-data/"
 DST="./modules/"
-
 os.mkdir(DST)
 
 for folders in  os.listdir(DIR):
```

Comment:
nitpick: typically the loop variable is named the singular while what it's looping through is called the plural version of whatever the subject is, i.e. in your case "folders"

Reply:
done